addon/components/bs-button-group.js

```diff
@@ -179,27 +154,28 @@ export default class ButtonGroup extends Component {
    * @param {*} value
    * @public
    */
-  onChange() {}
 
   @action
   buttonPressed(pressedValue) {
+    if (!this.args.onChange) {
+      return;
+    }
     let newValue;
 
-    if (this.isRadio) {
+    if (this.args.type === 'radio') {
       newValue = pressedValue;
     } else {
-      if (!isArray(this.value)) {
-        newValue = A([pressedValue]);
+      if (!isArray(this.args.value)) {
+        newValue = [pressedValue];
```

Comment:
Hm, yeah, you might be right. My first thought was that that does not really belong into the new Octane world. But indeed it could be a breaking change. I'll revert for now, and add a comment to change this for the next major version.